SampleTests/SampleTests.swift

```diff
@@ -17,11 +17,13 @@ class SampleTests: XCTestCase {
 
     override func tearDown() {
         // Put teardown code here. This method is called after the invocation of each test method in the class.
+        
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/kmpnz/Sample/blob/1bc4493b4e0a03f5421bf5979f31bd9f04f7379e/SampleTests/SampleTests.swift#L20"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>SampleTests.swift:20</code></del></td>
    </tr>
  </tbody>
</table>